src/fparser/two/utils.py

```diff
@@ -1022,28 +1070,57 @@ def _cmpkey(self):
 
 
 class STRINGBase(StringBase):
-    """
-::
-    <STRING-base> = <XYZ>
-    """
-    match = staticmethod(StringBase.match)
+    '''STRING-base is XYZ. Matches an upper case version of the input
```

Comment:
I don't understand: what does "STRING-base is XYZ" mean?

Reply:
I've updated the description.